doc/tutorials/run_kata_containers.rst

```diff
@@ -0,0 +1,329 @@
+#. Enable the ``macvtap`` function in ACRN kernel.
+
+   .. code-block:: none
+
+      $ cd acrn-kernel
+      $ cp kernel_config_sos .config
+      $ sed -i "s/# CONFIG_MACVLAN is not set/CONFIG_MACVLAN=y/" .config
+      $ sed -i '$ i CONFIG_MACVTAP=y' .config
+      $ make clean && make olddefconfig && make && sudo make modules_install INSTALL_MOD_PATH=out/
+
+   Login Service VM, use the new ACRN kernel:
+
+   .. code-block:: none
+
+      $ sudo mount /dev/sda1 /mnt
+      $ scp -r <user name>@<host address>:<your workspace>/acrn-kernel/arch/x86/boot/bzImage /mnt/
+      $ scp -r <user name>@<host address>:<your workspace>/acrn-kernel/out/lib/modules/* /lib/modules/
+      $ conf_file=`sed -n '$ s/default //p' /mnt/loader/loader.conf`.conf
+      $ kernel_img=`sed -n 2p /mnt/loader/entries/$conf_file | cut -d'/' -f4`
+      $ sed -i "s/$kernel_img/bzImage/g" /mnt/loader/entries/$conf_file
+      $ sync && umount /mnt && sudo reboot
+
+.. note:: So far the Kata-containers can only be supported on the SDC scenario hypervisor due to a
+   `kata runtime issue <https://github.com/kata-containers/runtime/issues/1785>`_.
+
+Set up Kata-runtime environment on Service VM
+*********************************************
+
+Install Kata-containers and its dependencies
+============================================
+
+#. Install bundles on the Service VM::
+
+   $ sudo swupd bundle-add git make c-basic go-basic containers-basic
+
+#. Use ``golang`` to get the following kata-containers repositories onto the Service VM::
+
+   $ go get -d -u github.com/kata-containers/runtime
+   $ go get -d -u github.com/kata-containers/shim
+   $ go get -d -u github.com/kata-containers/proxy
+   $ go get -d -u github.com/kata-containers/agent
+   $ go get -d -u github.com/kata-containers/osbuilder
+   $ go get -d -u github.com/kata-containers/packaging
+   $ cd $(go env GOPATH)/src/github.com/kata-containers/runtime && make && sudo -E PATH=$PATH make install
+   $ cd $(go env GOPATH)/src/github.com/kata-containers/proxy && make && sudo make install
+   $ cd $(go env GOPATH)/src/github.com/kata-containers/shim && make && sudo make install
+   $ cd $(go env GOPATH)/src/github.com/kata-containers/agent && make && sudo make install
+
+Configure Docker
+================
+
+The block-based driver like device-mapper is required for running Kata-containers on ACRN.
+
+Configure a Docker proxy
+------------------------
+
+If you use a proxy server, run the following commands as a shell script to configure Docker,
+refer to this `tutorial <https://docs.01.org/clearlinux/latest/tutorials/kata.html#run-kata-containers>`_
+for detailed information:
+
+.. code-block:: bash
+
+   export http_proxy=<your http proxy>:<port>
+   export https_proxy=<your https proxy>:<port>
+   sudo mkdir -p /etc/systemd/system/docker.service.d/
+   cat <<EOF | sudo tee /etc/systemd/system/docker.service.d/50-runtime.conf
+   [Service]
+   Environment="DOCKER_DEFAULT_RUNTIME=--default-runtime kata-runtime"
```

Comment:
I'd rather we do not make this the default runtime so let's remove this.

The `docker` command-line we use always specifies the runtime anyway so no need to duplicate this.

Reply:
Yes, remove the default runtime setup lines here.